integration/plugin/volumes/helpers_test.go

```diff
@@ -0,0 +1,67 @@
+package volumes
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/integration-cli/fixtures/plugin"
+	"github.com/docker/docker/pkg/locker"
+	"github.com/pkg/errors"
+)
+
+var pluginBuildLock = locker.New()
+
+func ensurePlugin(t *testing.T, name string) string {
```

Comment:
Would be nice to add a function comment about expectations:
"DummyPluginSrc at cmd/dummy
DummyPluginInstallPath at $GOPATH/bin/dummy"

Reply:
Not in this PR, but we should have a common set of helpers for all plugins, instead of duplicating code.